__tests__/app.test.js

```diff
@@ -111,4 +111,38 @@ describe('GEt /api/articles', () => {
             expect(body.articles).toBeSortedBy('created_at', { descending: true});
         });
     })
-})
+});
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200: responds with an array of comments for the given article_id', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .expect('Content-Type', /json/)
+        .then(({ body }) => {
+            expect(body.comments).toBeInstanceOf(Array);
+            expect(body.comments.length).toBeGreaterThan(0);
```

Comment:
Do we necessarily expect the length to be greater than 0, though? What if an article is valid, but has no comments yet - wouldn't we just expect an empty array (which would also be fine)?